### ISSUE TITLE
Add new description element (as a child of the result element)

### ISSUE THREAD

**Issue by dalepotter** [IATI/IATI-Schemas#360]
Similar to #357, add a new element `iati-activities/iati-activity/result/description` with definition:

> A longer, human-readable description.

The attribute will have an occurrence rule of `0..1`.

This will also add the following sub-attributes/elements with definitions:

- `iati-activities/iati-activity/result/description/narrative` - occurs `1..*`, type `xsd:string`
> The free text name or description of the item being described. This can be repeated in multiple languages. 

- `iati-activities/iati-activity/result/description/narrative/@xml:lang` - occurs `0..1`, 
> A code specifying the language of text in this element.It is recommended that wherever possible only codes from ISO 639-1 are used.

**Tasks:**

- [ ] Add tests
- [ ] Add `iati-activities/iati-activity/result/description`
- [ ] Add `iati-activities/iati-activity/result/description/narrative`
- [ ] Add `iati-activities/iati-activity/result/description/narrative/@xml:lang`
- [ ] Update Codelist mapping file for `iati-activities/iati-activity/result/description/narrative/@xml:lang` (to reference this attribute to the `Language` codelist)

**Comment by allthatilk** [comment]
This issue is likely based on a spreadsheet error. Flagged to BAs for review.

**Comment by PetyaKangalova** [comment]
@allthatilk @dalepotter @hayfield  I have just checked and there is no proposal for adding description at result level (as it has already been[ included at 2.02](http://iatistandard.org/202/activity-standard/iati-activities/iati-activity/result/)). Thanks for spotting @allthatilk 

It is basically an error in the spreadsheet (highlighted in green by mistake). I have now updated the [public spreadsheet](https://docs.google.com/spreadsheets/d/1XWnzesMvLebIVLk47OIXATfhcjESVcaGnw8SJpOuCCk/edit#gid=595223309) by removing the green color in the specific cells (row 353-355). I think the issue can be closed if you are all happy with that.
